pcsx2/gui/App.h

```diff
@@ -241,12 +242,12 @@ class pxAppResources
 public:
 	AppImageIds					ImageId;
 
-	ScopedPtr<wxImageList>		ConfigImages;
-	ScopedPtr<wxImageList>		ToolbarImages;
-	ScopedPtr<wxIconBundle>		IconBundle;
-	ScopedPtr<wxBitmap>			Bitmap_Logo;
-	ScopedPtr<wxBitmap>			ScreenshotBitmap;
-	ScopedPtr<AppGameDatabase>	GameDB;
+	std::unique_ptr<wxImageList>		ConfigImages;
+	std::unique_ptr<wxImageList>		ToolbarImages;
+	std::unique_ptr<wxIconBundle>		IconBundle;
+	std::unique_ptr<wxBitmap>			Bitmap_Logo;
+	std::unique_ptr<wxBitmap>			ScreenshotBitmap;
+	std::unique_ptr<AppGameDatabase>	GameDB;
 
 	pxAppResources();
```

Comment:
FWIW I just forced the tab indent size on user styles addon and it seems to be properly aligned now :) 

![pic](http://ultraimg.com/images/Screenshot23fe25.png)